doc/conf.py

```diff
@@ -65,7 +65,7 @@ def find_version(*file_paths):
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
+# The master doctree document.
```

Comment:
I don't think this is actually correct -- I believe this comment indicates that "index" is the root in teh table of contents.